.pre-commit-config.yaml

```
@@ -1,11 +1,11 @@
 repos:
 -   repo: https://github.com/phantomcyber/dev-cicd-tools
-    rev: v1.16
+    rev: v1.24
```

```suggestion
    rev: v1.26
```

We already have 1.26 available ;)